### PR TITLE
WIP: Updating to govcloud region via envrc

### PR DESCRIPTION
**Pull request by mr337**
## Description

Explain a little about the changes at a high level.

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
echo "Code goes here"
```

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](tbd) for this change
* [this article](tbd) explains more about the approach used.

## Screenshots

If this PR makes visible UI changes, an image of the finished UI can help reviewers and casual
observers understand the context of the changes. A before image is optional and
can be included at the submitter's discretion.

Consider using an animated image to show an entire workflow instead of using multiple images. You may want to use GIPHY CAPTURE for this! 📸

_Please frame screenshots to show enough useful context but also highlight the affected regions._

### CODE REVIEW

File: .envrc [hunk 1]
```diff
@@ -170,13 +170,13 @@ export TZ="UTC"
 #
 # Your AWS credentials should be setup in the transcom-ppp profile using
 # aws-vault. They will be detected and used by the app automatically.
-export AWS_S3_BUCKET_NAME="transcom-ppp-app-devlocal-us-west-2"
-export AWS_S3_REGION="us-west-2"
+export AWS_S3_BUCKET_NAME="transcom-ppp-app-devlocal-us-gov-west-1"
```
Comment (kilbergr) [inline .envrc:173]: Does this bucket exist? If so, in which account? `transcom-ppp` and `us-gov-west-1` seem opposed.

**Review by kilbergr — CHANGES_REQUESTED** [review]
I'm working on a ticket: https://github.com/transcom/transcom-infrasec-com/pull/1214/files
That is all about dealing with the repercussions of not having a single account anymore. Would love your input on that! But I think it might be more of a slower roll out than a quick fix.

File: .envrc (hunk 1 above)
Comment (mr337) [reply .envrc:173]: This was a victim of the blind replace. I'm guessing it probably doesn't. This is just a junk PR to show the changes to get you unblocked.

**Comment by mr337** [comment]
Going to close as this PR was just temporary.

**Pull request closed without merging**